core/src/main/java/com/sequenceiq/cloudbreak/core/bootstrap/service/host/ClusterHostServiceRunner.java

```diff
@@ -450,11 +451,13 @@ private void decoratePillarWithClouderaManagerCsds(Cluster cluster, Map<String,
     private void decoratePillarWithClouderaManagerSettings(Map<String, SaltPillarProperties> servicePillar, Long clusterId) {
         ClouderaManagerRepo clouderaManagerRepo = clusterComponentConfigProvider.getClouderaManagerRepoDetails(clusterId);
         boolean deterministicUidGid = isVersionNewerOrEqualThanLimited(clouderaManagerRepo.getVersion(), CLOUDERAMANAGER_VERSION_7_2_1);
+        boolean enableKnoxRangerAuthorizer = isVersionNewerOrEqualThanLimited(clouderaManagerRepo.getVersion(), CLOUDERAMANAGER_VERSION_7_2_0);
```

Comment:
This isn't a CM config. This should be moved to saveGatewayPillar` and consume it from there.

Reply:
@keyki I updated the PR with changes let me know what you think. I am new to this part of code so thanks for helping out 👍